### PR TITLE
Update genesis config

### PR DESCRIPTION
**Pull request by arminsabouri**
And massive rebase results

### CODE REVIEW

File: crates/primitives/res/genesis/botanix_testnet.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "alloc": {
+        "0x5c835BB1Af805341dBd37498C53AA0168B93582E": {
+            "balance": "0xD3C21BCECCEDA1000000"
+        },
+        "0xF27a6Ea4a1d5f7341Da7EDAaa47C5C933b738f4F": {
+            "balance": "0xD3C21BCECCEDA1000000"
+        },
+        "0x0Ea320990B44236A0cEd0ecC0Fd2b2df33071e78": {
+            "balance": "0xD3C21BCECCEDA1000000"
+        }
```
Comment (stevenroose) [inline crates/primitives/res/genesis/botanix_testnet.json:18]: Curious, what are these three addresses getting a whole bunch of coins?

File: crates/primitives/src/chain/mod.rs [hunk 2]
```diff
@@ -199,21 +205,23 @@ impl Default for Chain {
...
+        //     let chain = u.int_in_range(0..=(ethers_core::types::Chain::COUNT - 1))?;
 
-            return Ok(Chain::Named(ethers_core::types::Chain::iter().nth(chain).expect("in range")))
-        }
-
-        Ok(Self::Id(u64::arbitrary(u)?))
+        //     return Ok(Chain::Named(ethers_core::types::Chain::iter().nth(chain).expect("in
+        // range"))) }
+        /// TODO fix this
+        Ok(Self::Id(3000))
```
Comment (stevenroose) [inline crates/primitives/src/chain/mod.rs:219]: What does 3000 mean? Above botanix testnet is set to 444? If it was intended to be this, maybe you could `return Ok(Self::botanix_testnet())`? But I'm not following what this Arbitrary is..

Comment (arminsabouri) [reply crates/primitives/src/chain/mod.rs:219]: I would ignore this entirely this change dies with this PR and never got into main

File: crates/primitives/res/genesis/botanix_testnet.json (hunk 1 above)
Comment (arminsabouri) [reply crates/primitives/res/genesis/botanix_testnet.json:18]: Its Nick mostly.

Comment (arminsabouri) [reply crates/primitives/res/genesis/botanix_testnet.json:18]: Jk. One of them is "test" x 12. Another is the sidecar hot wallet. Last one is the genesis contract

**Pull request closed without merging**